heareval/tasks/util/audio.py

```diff
@@ -58,8 +65,8 @@ def resample_wav(in_file: str, out_file: str, out_sr: int):
             "-y",
             "-i",
             in_file,
-            "-af",
-            "aresample=resampler=soxr",
+            # "-af",
+            # "aresample=resampler=soxr",
```

Comment:
What's wrong with sox? Does this not work on your system?

Reply:
Yes. Some issue with configuring that in M1 mac. So I just remove the line, do the development and then uncomment.